etc/recipes/lsst-libffi/build.sh

```diff
@@ -0,0 +1,15 @@
+./configure --prefix=$PREFIX || exit 1
+make || exit 1
+make install || exit 1
```

Comment:
I *think* `conda-build` runs `build.sh` scripts with `-e`, so the explicit exits shouldn't be required.